src/components/challengeWidget/challengeWidget.js

```diff
@@ -3,27 +3,28 @@ import React from "react";
 
 import "./challengeWidget.css";
 
-const ChallengeWidget = ({ title, linkTo = "#" }) => (
+const ChallengeWidget = ({ title, label, linkTo = "#" }) => (
   <div className="challengeWidget">
     <Link to={linkTo}>
       <h1>{title}</h1>
```

Comment:
This should be an `<h2>` instead of an `<h1>` (as you should only have one `<h1>` element in a document)